configurations-conception-communes/fil-ariane.md

```diff
@@ -0,0 +1,243 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/breadcrumb-trail.html"
+breadcrumbs:
+  - title: "À propos de Canada.ca"
+    link:  "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: "Système de conception de Canada.ca"
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: "Bibliothèque de modèles et de configurations de conception"
+    link: "https://conception.canada.ca/configurations-conception-communes/connexion-contextuel.html"
```

Comment:
I don't think that's the right link

Reply:
I will review all the breadcrumbs.  Some of them might be a bit off.

Reply:
The breadcrumb links are all fixed.  We can discuss best way to implement.